sycl/include/sycl/kernel_bundle_enums.hpp

```diff
@@ -20,7 +20,13 @@ enum class bundle_state : char {
 
 namespace ext::oneapi::experimental {
 
-enum class source_language : int { opencl = 0, spirv = 1, sycl = 2 /* cuda */ };
+enum class source_language : int {
+  opencl = 0,
+  spirv = 1,
+  sycl = 2,
+  /* cuda */
+  sycl_jit /* temporary, alternative implementation for SYCL */
```

Comment:
Since it is temporary, I think the previous value given (`= 99`) is a good idea. Problem is, if someone compiles with this in use and it crosses the library boundary, if we then remove it and another different enum value is added, compatibility could be broken. It's unlikely to happen, but no impossible.

Reply:
Done.